### PR TITLE
Add test for bug1

### PR DESCRIPTION
**Pull request by wcandillon**

### CODE REVIEW

**Comment by wcandillon** [comment]
@MatthiasBrantner I added a test for the bug1 you sent. Doesn't fail. Doesn't fail if I do it manually as well.

**Comment by wcandillon** [comment]
@MatthiasBrantner my assumption is that it is a regression from the branch your testing. @dknochen which branch is it? At this point of the game, wouldn't be possible to use master for the demo?

File: tests/e2e/demo-spec.js [hunk 1]
```diff
@@ -39,8 +39,13 @@ describe('Report', function(){
         it('Creates a us-gaap:Assets synonym', function(){
             var synonyms = report.taxonomy.getSynonyms(conceptName);
             synonyms.get();
+            expect(synonyms.getSynonyms().count()).toBe(0);
             synonyms.addSynonym('us-gaap:Assets');
-            expect(synonyms.count()).toBe(1);
+            synonyms.addSynonym('us-gaap:AssetsCurrent');
+            synonyms.addSynonym('us-gaap:AssetsCurrent');
+            expect(synonyms.getSynonyms().count()).toBe(2);
```
Comment (wcandillon) [inline tests/e2e/demo-spec.js:46]: @MatthiasBrantner can you confirm that this is the expected result?

**Comment by MatthiasBrantner** [comment]
@wcandillon It's about adding a concept when the content of the text box was auto completed. In this case, it only adds the concept with the name up to what was actually typed and not the part that was auto completed.

**Comment by wcandillon** [comment]
@MatthiasBrantner what's text box autocomplete? I'm not familiar with that. I would need to investigate.

**Comment by MatthiasBrantner** [comment]
I have no idea. Maybe a Safari feature but clearly this feature works on other
websites.

File: tests/e2e/demo-spec.js (hunk 1 above)
Comment (dknochen) [inline tests/e2e/demo-spec.js:46]: @wcandillon this is a bug https://github.com/28msec/nolap-report-editor/issues/100 and should be fixed.

**Comment by wcandillon** [comment]
@MatthiasBrantner approves

**Pull request merged**